src/shape/compound.rs

```diff
@@ -51,6 +52,58 @@ impl<N: RealField> Compound<N> {
             nbits,
         }
     }
+
+    /// Creates a new 2D concave polygon from a set of points assumed to describe a
+    /// counter-clockwise convex polyline.
+    pub fn new_concave_polygon(isometry: Isometry<N>, hull: &[(N, N)]) -> Compound<N>
+    where
+        N: spade::SpadeFloat,
+    {
+        assert!(hull.len() >= 3, "A polygon must have at least 3 vertex");
+        assert!(
+            hull.first() == hull.last(),
+            "The hull must be closed (the first and last vertex be the same)"
+        );
+
+        let mut delaunay = FloatDelaunayTriangulation::with_walk_locate();
+
+        // Add each vertex of the hull one by one. An index (accessible with the `fix()` method is
+        // associated with each vertex. This index is strictly increasing.
+        for (x, y) in hull {
+            let _ = delaunay.insert([*x, *y]);
+        }
+
+        Compound::new(
+            delaunay
+                .triangles()
+                .filter_map(|face| {
+                    let indexes = {
+                        let vertex_handle = face.as_triangle();
+                        let mut indexes = [0; 3];
+                        for i in 0..3 {
+                            indexes[i] = vertex_handle[i].fix();
+                        }
+                        indexes
+                    };
```

Comment:
Note: this could be rewrote using the [collect_tuple](https://docs.rs/itertools/0.9.0/itertools/trait.Itertools.html#method.collect_tuple) method of itertools:

```
let (a, b, c) = face
    .as_triangle()
    .iter()
    .map(|vertex_handle| vertex_handle.fix())
    .collect_tuple()
    .unwrap();
```

But I don't know what is the best way to create an array by applying a function to another array.